engine/src/main/java/com/datatorrent/stram/LaunchContainerRunnable.java

```diff
@@ -172,6 +173,18 @@ public void run()
         if (archives != null) {
           addFilesToLocalResources(LocalResourceType.ARCHIVE, archives, localResources, fs);
         }
+        String configuredAppPath = dag.getValue(LogicalPlan.APPLICATION_PATH);
+        Path log4jPath = new Path(configuredAppPath + File.separator + StramClient.APEX_LOG4J_PROPS_FILE);
```

Comment:
changing name to log4j. properties. Still user won't get freedom to rename file present in conf dir.
If he really wants to rename, he can add renamed properties file to application jar and add -Dlog4j.configuration option to properties and it should work.